lerna.json

```diff
@@ -3,5 +3,5 @@
     "packages/*"
   ],
   "useNx": false,
-  "version": "1.1.4"
+  "version": "independent"
```

Comment:
what does independent do ?

Reply:
it's releases every package based on their own versions. otherwise everything we have a release it will change the other packages versions. that's why we have chaotic version list